Libraries/Lists/FlatList.js

```diff
@@ -457,13 +470,10 @@ class FlatList<ItemT> extends React.PureComponent<Props<ItemT>, void> {
   }
 
   _hasWarnedLegacy = false;
-  _listRef: VirtualizedList;
+  _listRef: null | VirtualizedList | ListView;
```

Comment:
Why would the listRef ever be a ListView?

Reply:
Right, it should be MetroListView instead (https://github.com/cdlewis/react-native/blob/22e2686a7ec2bc81656d8309676ddbd74057f0ba/Libraries/Lists/FlatList.js#L624). My bad for not noticing this, probably still passed flow because they have the same props.

Reply:
Hmmm, when I change it to MetroListView I see a bunch of errors related to that component not having properties such as `getScrollResponder`. Any ideas?

Reply:
Hmm if it's just `getScrollResponder` maybe you could just add it to MetroListView and just call the method on the underlying ListView ref.